test/store-listener-component-test.js

```diff
@@ -203,6 +255,18 @@ export default {
       assert(span.props.TestStore.x === 'foobar')
     },
 
+    'children props work with the transform function'() {
+      const node = TestUtils.renderIntoDocument(
+        <AltContainer stores={{ TestStore }} transform={transformMock}>
```

Comment:
i don't think you need the transformMock function at the top for this, just pass in test store and then check for it in the children props, no?